data/avro-schema/src/main/resources/avro/net/corda/data/crypto/ShortHashes.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "record",
+  "name": "ShortHashes",
+  "doc": "List of short hashes. A short hash is the first 12 characters of the hex string form of the hash",
```

Comment:
Please make this something like:

  List of short hashes. A short hash is the first 12 characters of the hex string form of the hash. Short hashes must only be used when arrangements have been made to handle collisions, whether accidental or malicious.